apps/changelog/commands.py

```diff
@@ -0,0 +1,63 @@
+# Third Party (PyPI) Imports
+import click
+
+# HTK Imports
+from htk.apps.changelog.classes import (
+    ChangeLog,
+    LogEntry,
+)
+from htk.apps.changelog.utils import (
+    fetch_git_logs,
+    fetch_origin_url,
+)
+
+
+def update_changelog_command_factory(slack_channel, changelog_file_name):
```

Comment:
# Reviewer Note
Name changed from `update_changelog_command_creator` to `update_changelog_command_factory`